libsolidity/analysis/ContractLevelChecker.cpp

```diff
@@ -273,51 +271,96 @@ void ContractLevelChecker::checkIllegalOverrides(ContractDefinition const& _cont
 			);
 	}
 
+	for (ModifierDefinition const* modifier: _contract.functionModifiers())
```

Comment:
Please add tests for the 3 error cases in this block.

Reply:
2 were already there, added another.